tests/tests_e3/os/process/main_test.py

```diff
@@ -17,6 +18,21 @@
     psutil = None
 
 
+def test_run_stdout_stderr(caplog):
+    """Verify that the parse shebang option works."""
+    prog_filename = os.path.join(os.getcwd(), "prog")
+    with open(prog_filename, "wb") as f:
+        f.write(b"#!/usr/bin/env python\n")
+        f.write(b"import sys\n")
+        f.write(b'print("stdout", file=sys.stdout)\n')
+        f.write(b'print("stderr", file=sys.stderr)\n')
+    e3.os.fs.chmod("a+x", prog_filename)
+    p = e3.os.process.Run([prog_filename], error=subprocess.PIPE)
+    assert p.out == "stdout\n"
```

Comment:
You can also assert that `p.err == "stderr"`. That's the purpose of `error=subprocess.PIPE`. Otherwise, it's being redirected to stdout (which is being checked here, but not below).